Board.java

```diff
@@ -1,42 +1,55 @@
 public class Board {
     private String state;
-    private int move;
+    private char lastTurn = 'o';
+    private int width;
 
-    public Board(String state, int move) {
+    static int[][] win = {{0, 1, 2}, {3, 4, 5}, {6, 7, 8}, {0, 3, 6}, {1, 4, 7}, {6, 7, 8}, {0, 4, 8}};
+
+    public Board(String state) {
         this.state = state;
-        this.move = move;
+        width = state.length();
```

Comment:
I think this would better be named `size` or `dimension`... since it's not just the width but also the height